### ISSUE TITLE
New release causes production site to be blank

### ISSUE THREAD

**Issue by sivayogasubramanian** [TEAMMATES/teammates#11875]
- Production

After a new release and deployment, going to [production site](https://teammatesv4.appspot.com/) causes a white blank screen.

**Steps to reproduce**

- Go to production after a new release. 

**Expected behaviour**

- Should show the website normally

**Actual behaviour**

- Shows white blank screen
- Only shows the screen after a hard refresh. We should not require users to do this.

**Additional info**

Possibly caused by some caching issues.

**Comment by damithc** [comment]
Yes, I too have noticed pages going fully or partially blank after a new release. And yes, I think it is a caching issue. A forced refresh usually fixes it.

**Comment by samuelfangjw** [comment]
Potentially resolved by removal of service worker in #13966.
